Makefile

```diff
@@ -1,16 +1,40 @@
+.PHONY: clean quality requirements validate
+
 clean:
-	find . -name '*.pyc' -delete
+	find . -name '__pycache__' -exec rm -rf {} +
+	find . -name '*.pyc' -exec rm -f {} +
+	find . -name '*.pyo' -exec rm -f {} +
+	find . -name '*~' -exec rm -f {} +	
 	coverage erase
 	rm -rf coverage htmlcov
+	rm -fr build/
+	rm -fr dist/
+	rm -fr *.egg-info
 
 quality:
 	tox -e quality
 
+
 requirements:
-	pip install -r test_requirements.txt
+	pip install -r requirements/dev.txt
 
 validate: clean
 	tox
+	
+selfcheck: ## check that the Makefile is well-formed
```

Comment:
Is this required by something?  It doesn't actually check anything.

Reply:
Ya, this is not necessary anymore, I used it for debugging sake. Will remove it